packages/commonwealth/server/routes/getThreads.ts

```diff
@@ -38,6 +38,10 @@ const getThreads = async (models: DB, req: Request, res: Response) => {
             },
           ],
         },
+        {
+          model: models.Comment,
```

Comment:
this created OOM errors.
https://github.com/hicommonwealth/commonwealth/issues/3950

As you mentioned on slack, you are just looking for comment count, we have util function already implemented here:
https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/server/util/getThreadCommentsCount.ts

Reply:
Hi, Thanks for the heads up @NakulManchanda. Will definitely give it a go today.

Reply:
update looks good to me, thanks!